client/adventureMap/AdventureMapShortcuts.cpp

```diff
@@ -36,6 +36,14 @@
 #include "../../lib/mapping/CMap.h"
 #include "../../lib/pathfinder/CGPathNode.h"
 
+bool isCurrentPlayerHuman()
+{
+	PlayerColor currentPlayer = LOCPLINT->cb->getCurrentPlayer();
+	bool isHuman = LOCPLINT->cb->getStartInfo()->playerInfos.count(currentPlayer)
+				   && LOCPLINT->cb->getStartInfo()->playerInfos.at(currentPlayer).isControlledByHuman();
+	return isHuman;
+}
+
```

Comment:
FYI - if you really want free function it must be declared as `static` - to have local visibility (this file only)